### PR TITLE
Dev center nav updates

### PR DESCRIPTION
**Pull request by timngyn**
*Description of changes:*
- Updated dev center nav look
  - Use media queries for mobile dev center nav
- Updated learn nav sizes

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/components/LearnNavBar/LearnNavBar.tsx [hunk 1]
```diff
@@ -1,14 +1,19 @@
 import { LearnNavBarDesktop } from "./LearnNavBarDesktop";
```
Comment (jacoblogan) [inline src/components/LearnNavBar/LearnNavBar.tsx:1]: Is this LearnNavBar being used? can it just be removed completely?

File: src/components/GlobalNav/GlobalNav.tsx [hunk 2]
```diff
@@ -23,15 +24,17 @@ export interface NavMenuItem {
 export interface NavProps {
   leftLinks: NavMenuItem[];
   rightLinks: NavMenuItem[];
+  socialLinks: NavMenuItem[];
   currentSite: string;
-  secondaryNavDesktop?: JSX.Element;
-  secondaryNavMobile?: JSX.Element;
+  secondaryNavDesktop: any;
```
Comment (jacoblogan) [inline src/components/GlobalNav/GlobalNav.tsx:29]: do we need to type these as `any`? It looks like we are always passing in `JSX.Element`.

File: src/components/GlobalNav/GlobalNav.tsx [hunk 3]
```diff
@@ -43,46 +46,112 @@ export function GlobalNav({
...
+            />
+          </Flex>
+          <RightNavLinks
+            rightLinks={rightLinks}
+            socialLinks={socialLinks}
+            currentSite={currentSite}
+            isCollapsed={isCollapsed}
+          />
+        </Flex>
+        <Flex
```
Comment (jacoblogan) [inline src/components/GlobalNav/GlobalNav.tsx:112]: Can we separate this out into its own component to make it easier to follow?

File: src/components/LearnNavBar/LearnNavBar.module.scss [hunk 4]
```diff
@@ -26,26 +31,32 @@
...
-  a {
-    color: var(--amplify-colors-neutral-40);
-  }
-
-  &:focus,
-  &:hover {
-    a {
-      color: var(--amplify-colors-neutral-10);
-    }
+// .mobile-nav-menu-items {
```
Comment (jacoblogan) [inline src/components/LearnNavBar/LearnNavBar.module.scss:41]: can we just remove these commented out styles

**Review by jacoblogan — DISMISSED** [review]
Nothing blocking just a few cleanup suggestions

File: src/components/LearnNavBar/LearnNavBar.tsx (hunk 1 above)
Comment (timngyn) [reply src/components/LearnNavBar/LearnNavBar.tsx:1]: Can be removed, thanks for calling it out. I definitely need to clean it up some more

**Review by jacoblogan — APPROVED** [review]

**Pull request merged**